tests/pylint/test_enforce_type_hints.py

```diff
@@ -565,3 +565,72 @@ async def async_lock(
 
     with assert_no_messages(linter):
         type_hint_checker.visit_classdef(class_node)
+
+
+def test_named_arguments(
+    linter: UnittestLinter, type_hint_checker: BaseChecker
+) -> None:
+    """Check missing entity properties when ignore_missing_annotations is False."""
+    # Set bypass option
+    type_hint_checker.config.ignore_missing_annotations = False
+
+    class_node, func_node, percentage_node, preset_mode_node = astroid.extract_node(
+        """
+    class FanEntity():
+        pass
+
+    class MyFan( #@
+        FanEntity
+    ):
+        async def async_turn_on( #@
+            self,
+            percentage, #@
+            preset_mode: str, #@
```

Comment:
```suggestion
            percentage, #@
            *,
            preset_mode: str, #@
```
Just to also test keyword_only args.

Reply:
Done